load_test.py

```diff
@@ -0,0 +1,74 @@
+import lumigator_demo as ld
+import pandas as pd
+import multiprocessing
+from loguru import logger
+from functools import partial
+
+# Global Vars
+
+TEAM_NAME = "stress_testers"
+DATASET_NAME = "thunderbird.csv"
+DATASET_ID = "db7ff8c2-a255-4d75-915d-77ba73affc53" # thunderbird dataset 
+
+# Evaluation
+MAX_SAMPLES=0
+EXPERIMENT_MULTIPLIER=3
+
+# Ground Truth 
+GPUS = 1
+REPLICAS = 2
+GT_PROCESS = 10 # number of concurrent ground truth generation runs
+
+## Run Ground Truth Generation 
+def process_ground_truth(index, deployment_id):
+    responses = []
+    # Load dataset of 10 rows into dataframe for processing
+    df = ld.dataset_download(DATASET_ID)
+    logger.info("Loaded dataset...")
+    for sample in df['examples'][:10]:
+        response = ld.get_bart_ground_truth(deployment_id, sample)
+        if response is not None:
+            responses.append((sample, response.text))
+    logger.info("Processed samples")
+
+## Run Evaluation
+def run_eval(max_samples=MAX_SAMPLES, experiment_multiplier=EXPERIMENT_MULTIPLIER):
+    """
+    max_samples: change the following to 0 to use all samples in the dataset (probably best for load testing)
+    experiment_multiplier: how many times you want to run each experiment
+    """
+
+    models = [
+        'hf://facebook/bart-large-cnn',
+        'hf://mikeadimech/longformer-qmsum-meeting-summarization',
+        'hf://mrm8488/t5-base-finetuned-summarize-news',
+        'hf://Falconsai/text_summarization',
+        "oai://gpt-3.5-turbo-0125", # smallest for load test
+        "mistral://open-mistral-7b"
+    ]
+
+    responses = []
+    for i in range(experiment_multiplier):
+        for model in models:
+            descr = f"Testing {model} summarization model on {DATASET_NAME}"
+            responses.append(ld.experiments_submit(model, TEAM_NAME, descr, DATASET_ID, max_samples))
```

Comment:
In case you added the HF format dataset as suggested above:

```suggestion
            responses.append(ld.experiments_submit(model, TEAM_NAME, descr, DATASET_ID_HF, max_samples))
```